### PR TITLE
Epmlstrtai 633

### PR DESCRIPTION
**Pull request by stelmakhmaksim**
Integrated frontend into backend. Setup Spring to serve static files.

### CODE REVIEW

File: src/main/java/com/epam/test_generator/services/jenkins/JenkinsServerFactoryImpl.java [hunk 1]
```diff
@@ -33,8 +33,8 @@ public JenkinsServer getJenkinsServer() {
     @Override
     @PostConstruct
     public void updateCredentials() {
-        this.defaultCredentials = new JenkinsCredentials("_",
-            "_", "_");
+        this.defaultCredentials = new JenkinsCredentials("http://ecse00100b3d.epam.com:8888/",
```
Comment (hdghg) [inline src/main/java/com/epam/test_generator/services/jenkins/JenkinsServerFactoryImpl.java]: move sensitive data out of the code. Use application.properties

File: src/main/java/com/epam/test_generator/controllers/GeneralController.java [hunk 2]
```diff
@@ -12,7 +12,7 @@
     @ApiOperation(value = "", hidden = true)
     @RequestMapping(value = "/")
     public String getMainPage() {
-        return "/newSuits";
+        return "index";
```
Comment (hdghg) [inline src/main/java/com/epam/test_generator/controllers/GeneralController.java]: Try to set project in a way this controller not required at all.

File: src/main/java/com/epam/test_generator/config/security/WebSecurityConfig.java [hunk 3]
```diff
@@ -108,8 +107,8 @@ public CorsConfigurationSource corsConfigurationSource() {
     @Override
     public void configure(WebSecurity web) throws Exception {
         web.ignoring().antMatchers("/login", "/registration", "/v2/api-docs",
-                "/configuration/ui", "/swagger-resources/**", "/configuration/**", "/swagger-ui.html",
-                "/webjars/**","/passwordForgot","/passwordReset","/confirmAccount");
+            "/configuration/ui", "/swagger-resources/**", "/configuration/**", "/swagger-ui.html",
+            "/webjars/**", "/passwordForgot", "/passwordReset", "/confirmAccount","/static/**","/index.html");
```
Comment (hdghg) [inline src/main/java/com/epam/test_generator/config/security/WebSecurityConfig.java]: Format code. colons must be followed by spaces

File: src/main/java/com/epam/test_generator/config/WebConfig.java [hunk 4]
```diff
@@ -102,11 +89,11 @@ public void addCorsMappings(CorsRegistry registry) {
      */
     @Override
     public void addResourceHandlers(ResourceHandlerRegistry registry) {
-        registry.addResourceHandler("/static/**")
-            .addResourceLocations("/WEB-INF/static/");
+        registry.addResourceHandler("/static/views/vue-static/**")
```
Comment (hdghg) [inline src/main/java/com/epam/test_generator/config/WebConfig.java]: I'd like to ask about frontend deployment strategy off line today after meeting

File: src/main/java/com/epam/test_generator/services/jenkins/JenkinsServerFactoryImpl.java (hunk 1 above)
Comment (hdghg) [reply src/main/java/com/epam/test_generator/services/jenkins/JenkinsServerFactoryImpl.java]: As a result of today's meeting we decided it is not necessary

File: src/main/java/com/epam/test_generator/config/security/WebSecurityConfig.java (hunk 3 above)
Comment (stelmakhmaksim) [reply src/main/java/com/epam/test_generator/config/security/WebSecurityConfig.java]: Fixed

File: src/main/java/com/epam/test_generator/controllers/GeneralController.java (hunk 2 above)
Comment (stelmakhmaksim) [reply src/main/java/com/epam/test_generator/controllers/GeneralController.java]: General controller was deleted

**Pull request merged**